packages/core/src/index.tsx

```diff
@@ -19,6 +19,13 @@ if (
 ) {
     registerServiceWorker();
 }
+
+// if we don't clear the local storage, then exchange_rates subscription calls won't be made when user refreshes the page
+// check packages/stores/src/providers/ExchangeRatesProvider.tsx
+
+if (window.localStorage.getItem('exchange_rates')) {
+    window.localStorage.removeItem('exchange_rates');
+}
```

Comment:
Should this be in an useEffect ?

Reply:
I don't think so. it happens before the react app starts